infrastructure/Makefile

```diff
@@ -5,6 +5,10 @@ create_ecs_cluster:
 create_ecs_roles:
 	aws cloudformation create-stack --stack-name worker-management-ecs-iam --template-body file://03_roles.yaml --profile sinfra --region us-east-1 --capabilities CAPABILITY_IAM
 create_service:
-	aws cloudformation create-stack --stack-name worker-management-ecs-service --template-body file://04_service.yaml --profile sinfra --region us-east-1
+	aws cloudformation create-stack --stack-name worker-management-information-sys --template-body file://04_service.yaml --profile sinfra --region us-east-1
```

Comment:
@RohitRox can we not use a variable like `AWS_PROFILE:=sinfra`  and use it later in the file as:
```
create_ecr:
	aws cloudformation create-stack --stack-name worker-management-ecr --template-body file://01_ecr.yaml --profile $(AWS_PROFILE) --region us-east-1
```

Reply:
YES..let's merge this first...we need to change that and few other stuff too